rke2/ayekan/exporters/snmp-generator/Makefile

```diff
@@ -18,43 +18,28 @@ MIB_PATH := 'mibs'
-CISCO_URL           := https://raw.githubusercontent.com/cisco/cisco-mibs/2d465cce2de4e67a3561d8e41e4c99b597558d4b/v2
+CISCO_URL           := https://raw.githubusercontent.com/cisco/cisco-mibs/05dbf50226f7df5f52fd2dd1a9c17759273fa0d0/v2/
+CISCO_URL2           := https://raw.githubusercontent.com/cisco/cisco-mibs/05dbf50226f7df5f52fd2dd1a9c17759273fa0d0/v1/
 DELL_URL            := https://dl.dell.com/FOLDER11196144M/1/Dell-OM-MIBS-11010_A00.zip
 IANA_CHARSET_URL    := https://www.iana.org/assignments/ianacharset-mib/ianacharset-mib
 IANA_IFTYPE_URL     := https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib
 IANA_PRINTER_URL    := https://www.iana.org/assignments/ianaprinter-mib/ianaprinter-mib
-KEEPALIVED_URL      := https://raw.githubusercontent.com/acassen/keepalived/v2.2.8/doc/KEEPALIVED-MIB.txt
 VRRP_URL            := https://raw.githubusercontent.com/acassen/keepalived/v2.2.8/doc/VRRP-MIB.txt
 VRRPV3_URL          := https://raw.githubusercontent.com/acassen/keepalived/v2.2.8/doc/VRRPv3-MIB.txt
-KEMP_LM_URL         := https://kemptechnologies.com/files/packages/current/LM_mibs.zip
 MIKROTIK_URL        := 'https://box.mikrotik.com/f/a41daf63d0c14347a088/?dl=1'
 NEC_URL             := https://jpn.nec.com/univerge/ix/Manual/MIB
-NET_SNMP_URL        := https://raw.githubusercontent.com/net-snmp/net-snmp/v5.9/mibs
-PALOALTO_URL        := https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/zip/snmp-mib/pan-10-0-snmp-mib-modules.zip
-PRINTER_URL         := https://ftp.pwg.org/pub/pwg/pmp/mibs/rfc3805b.mib
-SERVERTECH_URL      := 'https://cdn10.servertech.com/assets/documents/documents/817/original/Sentry3.mib'
-SERVERTECH4_URL     := 'https://cdn10.servertech.com/assets/documents/documents/815/original/Sentry4.mib'
-SYNOLOGY_URL        := 'https://global.download.synology.com/download/Document/Software/DeveloperGuide/Firmware/DSM/All/enu/Synology_MIB_File.zip'
+NET_SNMP_URL        := https://raw.githubusercontent.com/net-snmp/net-snmp/refs/heads/master/mibs/
 UBNT_AIROS_URL      := https://dl.ubnt.com/firmwares/airos-ubnt-mib/ubnt-mib.zip
 UBNT_AIRFIBER_URL   := https://dl.ubnt.com/firmwares/airfiber5X/v4.1.0/UBNT-MIB.txt
 UBNT_DL_URL         := http://dl.ubnt-ut.com/snmp
 RARITAN_URL         := https://cdn.raritan.com/download/PX/v1.5.20/PDU-MIB.txt
 RARITAN2_URL        := https://cdn1.raritan.com/download/src-g2/4.0.20/PDU2_MIB_4.0.20_49038.txt
-INFRAPOWER_URL      := https://www.austin-hughes.com/wp-content/uploads/2021/05/IPD-03-S-MIB.zip
 LIEBERT_URL         := https://www.vertiv.com/globalassets/documents/software/monitoring/lgpmib-win_rev16_299461_0.zip
-READYNAS_URL        := https://www.downloads.netgear.com/files/ReadyNAS/READYNAS-MIB.txt
-READYDATAOS_URL     := https://www.circitor.fr/Mibs/Mib/R/READYDATAOS-MIB.mib
-XUPS_URL            := https://www.circitor.fr/Mibs/Mib/X/XUPS-MIB.mib
+XUPS_URL            := https://mibbrowser.online/mibs/XUPS-MIB.mib
```

Comment:
Should we try to contribute this upstream?

Reply:
I dont think that's the spirit of the upstream. Not all mibs are opensource and the maintainers have said that. There are literally thousands of mibs available, they only maintain a basic list in the makefile, so no, we shouldn't contribute to upstream since we could hit licensing issues.